Source/SocketIO/Client/SocketIOClient.swift

```diff
@@ -212,6 +212,19 @@ open class SocketIOClient: NSObject, SocketIOClientSpec {
     /// - parameter items: The items to send with this event. May be left out.
     /// - parameter completion: Callback called on transport write completion.
     open func emit(_ event: String, _ items: SocketData..., completion: (() -> ())? = nil)  {
+        emit(event, items, completion: completion)
+    }
+    
+    /// Send an event to the server, with optional data items and optional write completion handler.
+    ///
+    /// If an error occurs trying to transform `items` into their socket representation, a `SocketClientEvent.error`
+    /// will be emitted. The structure of the error data is `[eventName, items, theError]`
+    ///
+    /// - parameter event: The event to send.
+    /// - parameter items: The items to send with this event. May be left out.
+    /// - parameter completion: Callback called on transport write completion.
+    open func emit(_ event: String, _ items: [SocketData], completion: (() -> ())?) {
```

Comment:
Could you add `withItems` external param?

Reply:
Wouldn’t it be more “swifty” to do `open func emit(_ event: String, with items: [SocketData], completion: (() -> ())?)` then?

Reply:
Yeah that'd also be fine

Reply:
Done!